src/main/java/racingcar/model/Winner.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winner {
+
+    private List<Car> cars;
+
+    public Winner(List<Car> car) {
+        this.cars = car;
+    }
+
+    public List<String> getWinner() {
+        List<String> winners = new ArrayList<>();
+        int maxPosition = getMaxPosition();
```

Comment:
순서를 바꾸는 것이 더 좋을 것 같아요! 😃
```suggestion
        int maxPosition = getMaxPosition();
        List<String> winners = new ArrayList<>();
```